provisioning/device/tests/RegistrationOperationStatusTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices.Provisioning.Client.UnitTests
+{
+    [TestClass]
+    [TestCategory("Unit")]
+    public class RegistrationOperationStatusTests
+    {
+        private const string FakeOperationId = "testing-operation-id";
+
+        private static readonly ProvisioningRegistrationStatus s_status = ProvisioningRegistrationStatus.Assigned;
+        private static readonly DeviceRegistrationResult s_registrationState = new()
+        {
+            RegistrationId = "testing-registration-id",
+            CreatedOnUtc = DateTimeOffset.MinValue,
+            AssignedHub = "testing-iot-hub",
+            DeviceId = "testing-device-id",
+        };
+
+        [TestMethod]
+        public void RegistrationOperationStatus_Properties()
+        {
+            // arrange
+
+            var source = new RegistrationOperationStatus
+            {
+                OperationId = FakeOperationId,
+                Status = s_status,
+                RegistrationState = s_registrationState,
+            };
+            string body = JsonConvert.SerializeObject(source);
+
+            Console.WriteLine(body);
```

Comment:
Do you want to leave this in?

Reply:
Oh, I must forget to delete this after debugging. Good catch